JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -363,5 +363,16 @@ module Microsoft.ApplicationInsights {
             
             return true;
         }
+
+        public static getAbsoluteUrl = (function () {
+            var a;
+
+            return function (url) {
+                if (!a) a = document.createElement('a');
```

Comment:
To be clear it's wouldn't affect minimization, but it does make code cleaner. Thanks!